cwltool/sandboxjs.py

```diff
@@ -1,12 +1,11 @@
-import cStringIO
```

Comment:
well, it is not required to run the code, only to validate the types. But there are many more type violations due to missing imports in the code... so what should we do?